Mathlib/Tactic/Common.lean

```diff
@@ -112,3 +104,27 @@ import Mathlib.Util.AssertExists
 import Mathlib.Util.CountHeartbeats
 import Mathlib.Util.Imports
 import Mathlib.Util.WhatsNew
+
+/-!
+This file imports all tactics which do not have significant theory imports,
+and hence can be imported very low in the theory import hierarchy,
+thereby making tactics widely available without needing specific imports.
+
+We include some commented out imports here, with an explanation of their theory requirements,
+to save some time for anyone wondering why they are not here.
+-/
+
+/-!
+# Register tactics with `hint`.
+-/
+
+section Hint
+
+register_hint split
+register_hint intro
+register_hint aesop
+register_hint decide
+register_hint simp_all?
+register_hint exact?
+
```

Comment:
```suggestion
register_hint exact?
register_hint constructor

```
I think that Lean 3 had it and in Lean 4 is maybe more useful.
I wonder whether `split_ifs` is also an option.

Reply:
At present there is no way to remove a `register_hint` (without PRing to Mathlib), but it is easy to add.

Can I suggest that intentionally undershoot on this PR, and then let people try it out and revise the list?